### PR TITLE
added funky colors to the python game of life

### PR DESCRIPTION
**Pull request by upint-0**
<3

### CODE REVIEW

File: src/space_shit.tab
Comment (loleloup) [inline src/space_shit.tab:1]: remove this file, this is the file used to save a table

File: src/Tablewidget.py [hunk 1]
```diff
@@ -56,7 +60,9 @@ def paintEvent(self, event):
         for i in range(self.table.height):
             for j in range(self.table.width):
                 if self.table.table[i][j]:
-                    qp.setBrush(QtGui.QBrush(QtGui.QColor("black")))
+                    this_color = QtGui.QColor("white")
+                    this_color.setHsv(((self.step * 20 + i + j) * 10) % 255, 255, 255)
```
Comment (loleloup) [inline src/Tablewidget.py]: not sure what you do here, what are the number 20?  10?

File: src/space_shit.tab
Comment (upint-0) [reply src/space_shit.tab:1]: suck my ass, diy bitch

File: src/Tablewidget.py [hunk 2]
```diff
@@ -34,6 +34,10 @@ def get_height(self):
 
 class TableWidget(QtWidgets.QWidget):
 
+    colors = ("blue", "red", "yellow", "green")
+    color = "blue"
+    step = 0
```
Comment (loleloup) [inline src/Tablewidget.py]: step should be initialized in the __init__ method, not there

File: src/Tablewidget.py [hunk 3]
```diff
@@ -34,6 +34,10 @@ def get_height(self):
 
 class TableWidget(QtWidgets.QWidget):
 
+    colors = ("blue", "red", "yellow", "green")
+    color = "blue"
```
Comment (loleloup) [inline src/Tablewidget.py]: unused?

File: src/Tablewidget.py [hunk 4]
```diff
@@ -34,6 +34,10 @@ def get_height(self):
 
 class TableWidget(QtWidgets.QWidget):
 
+    colors = ("blue", "red", "yellow", "green")
```
Comment (loleloup) [inline src/Tablewidget.py]: unused?

**Review by loleloup — APPROVED** [review]

**Pull request merged**